src/api/app/components/notification_action_bar_component.html.haml

```diff
@@ -5,9 +5,8 @@
         .custom-control.custom-checkbox
           = check_box_tag('select-all-notifications', 1, false, class: 'custom-control-input')
           %label.custom-control-label.align-middle.mr-4#select-all-label{ for: 'select-all-notifications' } Select All
-          :ruby
-            button_text = type == 'read' ? "Mark as 'Unread'" : "Mark as 'Read'"
-            disable_with_content = button_text + tag.i(class: 'fas fa-spinner fa-spin ml-2')
           = button_tag(type: 'submit', class: 'btn btn-sm btn-outline-success px-3', id: 'done-button', disabled: 'disabled',
                        'data-disable-with': disable_with_content) do
             = button_text
+          = link_to(button_text(all: true), @update_path, method: :put, remote: true,
```

Comment:
Some design feedback. It's never such a good idea to have two call-to-actions buttons on one page.

Also the difference is very subtle (basically one word) and for less than 300 Notifications the buttons do exactly the same.

![Screenshot from 2021-10-05 12-12-34](https://user-images.githubusercontent.com/514785/136004263-bc4aac0a-517a-438d-a2c0-fc5ac41fb8ad.png)

Reply:
@hennevogel do we need to update anything? Or do you have any suggestions that could make it better?

Reply:
Some ideas:

- Use different colors for the buttons or don't use colors at all. The actions are equally "important".
- Display `Mark all as read` only if there are more Notifications than currently on the page
- Change `Mark as read` to `Mark selected as read`

Reply:
@hennevogel it's updated. 

![Screenshot_2021-10-05 Screenshot](https://user-images.githubusercontent.com/12820609/136034428-64b82639-5841-4382-a5b0-d61585eecb7f.png)